.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js: 10
+cache: yarn
+install:
+  - yarn
+matrix:
+  include:
+    - name: 'Backend'
+      before_install:
+        - cd server
+      script:
+        - yarn lint
```

Comment:
should we also add test/build for backend?

Reply:
yes, we should. but I don't know what to put for the test/build for backend. so I didn't add these scripts yet. I was planning to figure this out later. but if you can help us figure it out, that'll be great!

Reply:
@ifhuang Yup we should, I will add it in when it's ready! I was looking at #20 `server/package.json` file and it doesn't look like the test script is up, so I left it out for now so that it doesn't break anything when this is merged :p